applicationbase/bootstrap/keys.go

```diff
@@ -89,8 +90,14 @@ func createKeysInDir(
 			return nil, errors.Wrap(err, "[ createKeysInDir ] couldn't marshal keys")
 		}
 
-		inslogger.FromContext(ctx).Info("Genesis write key " + filepath.Join(dir, keyname))
-		err = makeFileWithDir(dir, keyname, result)
+		if properName {
```

Comment:
You can di like this
if properName {
    keyname = n.CertName
}
and then log and makeFileWithDir once

Reply:
Nice! Thanks

Reply:
Done